src/components/admin-dashboard-components/PastEvents/PastEvents.module.css

```diff
@@ -17,8 +17,8 @@ h5 {
   font-size: 14px;
   line-height: 19px;
   letter-spacing: -0.02em;
+  color: #115740;
```

Comment:
this should be referencing `var(--eden)` from global.css instead of hardcoding the color